modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-web/src/main/java/com/liferay/multi/factor/authentication/web/internal/frontend/taglib/servlet/taglib/MFASetupUserAccountScreenNavigationCategory.java

```diff
@@ -37,24 +30,19 @@
  */
 @Component(
 	property = "screen.navigation.category.order:Integer=40",
-	service = {ScreenNavigationCategory.class, ScreenNavigationEntry.class}
+	service = ScreenNavigationCategory.class
 )
 public class MFASetupUserAccountScreenNavigationCategory
-	implements ScreenNavigationCategory, ScreenNavigationEntry<User> {
+	implements ScreenNavigationCategory {
 
 	@Override
 	public String getCategoryKey() {
 		return MFASetupUserAccountScreenNavigationConstants.CATEGORY_KEY_MFA;
 	}
 
-	@Override
-	public String getEntryKey() {
-		return null;
```

Comment:
@RenanUmV It seems the MFA `Entry` services are registered on demand in `MFASetupUserAccountCheckerTracker`, but I'm not sure if this empty `Entry` implementation affects the UI.

Could you ask QA to check whether the UI still works as expected (no change with this PR)?